### ISSUE TITLE
Switch Storybook builds from PR comments to a Status Check?

### ISSUE THREAD

**Issue by lmorchard** [mozilla/fxa#5474]
Storybook builds are noisy in comments. This is mainly because it's simple to post a comment and I already knew how to do it 😅 

We could research & implement [GitHub Status Checks](https://developer.github.com/v3/repos/statuses/) so that Storybook builds appear in the status block at the bottom along with other things like CircleCI status. Seems like creating a status is as easy as a comment, but I've never tried it before

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1996)
┆Issue Number: FXA-1996

**Comment by LZoog** [comment]
+1, I like this idea a lot.

**Comment by lmorchard** [comment]
A downside to Status Checks is that I don't think they offer history. So, if there are changes from one commit to another in a PR in response to a review, I'm not sure there's an easy way to compare. Might be an enhancement for the little static site generator, though, since I just have PR index.html redirect to the latest commit

**Comment by lmorchard** [comment]
So, I *thought* this would be fast, so I went ahead and submitted a draft PR: https://github.com/mozilla/fxa/pull/5478

Hit a snag, though: The FxaStorybooks user can post comments to the mozilla/fxa repo without needing to be added as a contributor. Posting a Status Check, though, requires that the FxaStorybooks user be invited to the repo. That's probably not the end of the world, though I did want to keep that user isolated somewhat.
